cfbs/cfbs_config.py

```diff
@@ -298,9 +299,15 @@ def add_command(
         else:
             self._add_modules(to_add, added_by, checksum)
 
-        if len(self["build"]) == before_adding:
-            # Not an error, we just want to exit successfully without
-            # making a git commit
-            raise CFBSReturnWithoutCommit(0)
-
-        return 0
+        msg = ""
+        count = 0
+        for module in self["build"]:
+            if not any(module["name"] == m["name"] for m in before):
+                msg += "\n - Added module '%s'" % module["name"]
```

Comment:
I'd prefer a set difference to be used here, but YMMV.

Reply:
Like this?
```python
for name in {x["name"] for x in self["build"]}.difference(x["name"] for x in before):
    msg += "\n - Added module '%s'" % name
    count += 1
```
I think the current one is easier to read.

The following does not work:
```python
for module in set(self["build"]).difference(set(before)):
    msg += "\n - Added module '%s'" % module["name"]
    count += 1
```
```
Traceback (most recent call last):
...
  File "/usr/local/lib/python3.8/dist-packages/cfbs/cfbs_config.py", line 306, in add_command
    for module in set(self["build"]).difference(set(before)):
TypeError: unhashable type: 'collections.OrderedDict'
```

Reply:
I find the (working) set difference easiest to read only I would actually assign the two sets to variables like `module_names` and `orig_module_names`.

Reply:
I agree, thanks :rocket: